app/models/status_message.rb

```diff
@@ -120,6 +89,29 @@ def post_location
     }
   end
 
+  def people_allowed_to_be_mentioned
+    @aspects_ppl ||=
+      if public?
+        "all"
+      else
+        [
+          *AspectMembership.where(aspect_id: aspects).map(&:person),
```

Comment:
This line creates a DB-query for each person in this aspects, but this can be done with an all-in-one query:
```ruby
Contact.joins(:aspect_memberships).where(aspect_memberships: {aspect: aspects}).distinct.pluck(:person_id)
```